### PR TITLE
[`PEFT` / `docs`] Add a note about torch.compile

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

addresses: https://github.com/huggingface/diffusers/pull/6190#issuecomment-1923770278 

cc @sayakpaul

### CODE REVIEW

File: docs/source/en/tutorials/using_peft_for_inference.md [hunk 1]
```diff
@@ -165,6 +165,10 @@ list_adapters_component_wise
 {"text_encoder": ["toy", "pixel"], "unet": ["toy", "pixel"], "text_encoder_2": ["toy", "pixel"]}
 ```
 
+## Compatibility with `torch.compile`
+
+If you want to compile your model with `torch.compile` make sure to first fuse the LoRA weights into the base model as explained in the section below.
```
Comment (sayakpaul) [inline docs/source/en/tutorials/using_peft_for_inference.md:170]: Can we maybe show a working example too? That's better no?

**Review by sayakpaul — COMMENTED** [review]
Thanks! Just one comment. LMK.

**Comment by younesbelkada** [comment]
Thanks for the suggestion ! Done ! @sayakpaul

**Review by sayakpaul — APPROVED** [review]
Perfect!

**Pull request merged**